src/.nuget/Microsoft.NETCore.Jit/win/Microsoft.NETCore.Jit.pkgproj

```diff
@@ -9,9 +9,13 @@
   </PropertyGroup>
   <ItemGroup>
     <ArchitectureSpecificNativeFile Include="$(BinDir)clrjit.dll" />
+    <CrossArchitectureSpecificNativeFile Include="$(BinDir)$(CrossTargetComponentFolder)\clrjit.dll" />
     <File Include="@(ArchitectureSpecificNativeFile)">
       <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
     </File>
+    <File Condition="'$(HasCrossTargetComponents)' == 'true'" Include="@(CrossArchitectureSpecificNativeFile)">
+      <TargetPath>runtimes/$(CrossTargetComponentFolder)_$(PackagePlatform)/native</TargetPath>
```

Comment:
Agreed. Where you place the binary does also have an impact when you issue commands such as restore & publish. Binaries in tools folder does not get copied to target folder when you publish as it is not required at runtime. Hence it is important to get this right. I am not sure what the correct folder for this would be but if it not going to be required at runtime it should not be in runtimes folder.